docs/user/alerting/stack-alerts/es-query.asciidoc

```diff
@@ -31,7 +31,7 @@ Time window:: This clause determines how far back to search for documents, using
 [float]
 ==== Action variables
 
-When the ES query alert condition is met, the following variables are available to use inside each action:
+When the ES query alert condition is met, the following variables are available to each action, in addition to the <<defining-alerts-actions-variables, variables common to all alerts>>:
```

Comment:
Add an action to run with the alert condition is met.   The following variables are specific to ES queries. You can also specific variables common to all alerts.

Not sure if I understood the last part correctly.